app/models/suggested_councillor.rb

```diff
@@ -2,4 +2,6 @@ class SuggestedCouncillor < ActiveRecord::Base
   has_one :authority, through: :councillor_contribution
   belongs_to :councillor_contribution
   validates :councillor_contribution, :name, :email, presence: true
+  validates_email_format_of :email, :message => "format seems not right.
+  Does it follow the format of example@email.com?", :unless => Proc.new{|c| c.email.blank?}
```

Comment:
> My question now is, when I have multpiple validations for one field, how will I know which error message to display

I think for this first pass on validations it's fine to show multiple messages. We can refine the display as a follow up issue and see what the people using it find helpful.

Is that helpful?